src/link/internal.tsx

```diff
@@ -91,7 +91,8 @@ const InternalLink = React.forwardRef(
         styles[getFontSizeStyle(variant, fontSize)],
         styles[getColorStyle(variant, color)]
       ),
-      'aria-label': ariaLabel,
+      'aria-label':
+        ariaLabel && external && externalIconAriaLabel ? ariaLabel + ', ' + externalIconAriaLabel : ariaLabel,
```

Comment:
Can you use `joinStrings` helper here? https://github.com/cloudscape-design/components/search?q=joinStrings

Is the comma required?

Reply:
Why the `externalIconAriaLabel` is not used if ariaLabel is not defined?

I would rather imagine:

```
if (ariaLabel && external && externalIconAriaLabel) {
  return ariaLabel + externalIconAriaLabel
} else if (ariaLabel) {
  return ariaLabel
} else if (external && externalIconAriaLabel) {
  return externalIconAriaLabel
}
```

Reply:
@just-boris  nvda and VO make a pause at the comma, so I think it is necessary.

@pan-kot Only when aria-label is defined on parent <a>, externalIconAriaLabel which is defined in child span, will be overriden.